### ISSUE TITLE
[Excalibur] Create R chart to show change in rate of transmission over time

### ISSUE THREAD

**Issue by rocktacular** [Recidiviz/covid19-dashboard#239]
As per design:
1. show initial/original R0
2. show current R
3. show changes over time 
4. show `y=1` line
5. tooltip to display R(t) value on hover

Questions: 
1. where are the historical data for this calculation? will it only be a straight line between 2 points?
2. is R(t) exposed on the model engine we have?

**Updated requirements:** 
Because R(t) can only be modelled on a per-facility basis and does not make sense to calculate for the system as a whole, we will show some summary statistics about the R(t) across the facilities:

1. X of Y facilities currently have an R(t) < 1
2. Average r(t) reduction across facilities: E.g., if f1 went from 3.7→ 1.7 and f2 went from 3.0→ 2.0, the average change across those facilities would be facility r(t) reduced by an average of 1.5
3. Max r(t) change: Difference between r(t)s between the facility with the original max r(t) and the facility with the current max r(t) (e.g., if original r(t)s were f1=5 and f2=3, and currently we have f1=2 and f2=2.5, then the difference would be 5-2.5 = max r(t) reduced by 2.5)

**Comment by jessex** [comment]
You should check with @macfarlandian that this is not the R(t) chart he has already started building.

**Comment by macfarlandian** [comment]
yeah same chart. part of #201 but not broken out as a separate issue.

**Comment by daschi** [comment]
after chatting with @macfarlandian and @sychang, will wait on working on this until we have an updated UI / decision on what data to display

**Comment by daschi** [comment]
Updated mock:
![Screenshot 2020-05-04 09 43 17](https://user-images.githubusercontent.com/5402804/80990748-b7e09080-8deb-11ea-8a7a-8dfd5b941042.png)
